From ebe473696da4f4de48b973c93c8bd32300baa818 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <b38018@freescale.com>
Date: Wed, 24 Aug 2011 16:11:14 -0500
Subject: [PATCH] egalax_ts: enable single event support

* Enable single event support for Ubuntu and
  Sato desktop touchscreen support

Signed-off-by: Adrian Alonso <b38018@freescale.com>
---
 drivers/input/touchscreen/egalax_ts.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/input/touchscreen/egalax_ts.c b/drivers/input/touchscreen/egalax_ts.c
index 58273b9..05c9c49 100644
--- a/drivers/input/touchscreen/egalax_ts.c
+++ b/drivers/input/touchscreen/egalax_ts.c
@@ -57,6 +57,9 @@
 
 #define MAX_I2C_DATA_LEN	10
 
+/* Enable single touch event mode */
+#define FORCE_SINGLE_POINTER_SUPPORT 1
+
 struct egalax_pointer {
 	bool valid;
 	bool status;
-- 
1.7.4.1

